umi_tools/group.py

```diff
@@ -403,7 +440,7 @@ def main(argv=None):
                         else:
                             # Add the 'UG' tag to the read
                             read.tags += [('UG', unique_id)]
-                            read.tags += [('FU', top_umi)]
+                            read.tags += [('BX', top_umi)]
```

Comment:
No I'm not sure about how Picard works with UMIs. I was taking this from this forum 
http://gatkforums.broadinstitute.org/gatk/discussion/7497/picard-markduplicates-with-umi
I'll run Picard and check the output...